package.json

```diff
@@ -39,21 +39,26 @@
     "discord-oauth2": "^2.11.0",
     "dotenv": "^16.0.3",
     "dotenv-flow": "^3.2.0",
+    "envalid": "^7.3.1",
     "express": "^4.18.2",
     "express-async-router": "^0.1.15",
+    "express-response-errors": "^1.0.5",
```

Comment:
[npm link](https://www.npmjs.com/package/express-response-errors)

This lets us do something like the following:
```js
throw new BadRequestError('missing name');
```

instead of this:
```js
return res.status(400).send({ message: 'missing name' })
```